galasa-ui/src/components/test-runs/TestRunsContent.tsx

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+'use client';
+
+import styles from "@/styles/TestRunsContent.module.css";
+
+export default function TestRunsContent() {
+  return (
+    <div className={styles.testRunsContentWrapper}>
+      <p className={styles.underConstruction}>
+        This page is under construction. Please come back later to query a list of test runs.
+      </p>
+    </div>
+  );
```

Comment:
This is great, but I don't think we need a separate TestRunsContent component at the moment. We can directly place this `div` into the `TestRunsPage` for simplicity, like:

```ts
export default function TestRunsPage() {
  return (
    <main id="content">
      <BreadCrumb />
      <PageTile title={"Test Runs"} />
      <div className={styles.testRunsContentWrapper}>
        <p className={styles.underConstruction}>
          This page is under construction. Please come back later to query a list of test runs.
        </p>
      </div>
    </main>
  );
```